hs_core/views/resource_rest_api.py

```diff
@@ -265,7 +270,7 @@ def get(self, request, pk):
             bag_url = site_url + reverse('download_refts_resource_bag',
                                          kwargs={'shortkey': pk})
         else:
-            bag_url = site_url + AbstractResource.bag_url(pk)
+            bag_url = site_url + AbstractResource.bag_url(pk) + '/true'
```

Comment:
@hyi It seems passing the value (```true```) for the parameter ```rest_call``` in this way is not correct. That's the reason the unit test is failing. Try the following approach: (ref: https://docs.djangoproject.com/en/1.10/ref/urlresolvers/#resolve)

```match = resolve(AbstractResource.bag_url(pk))```
```match.kwargs['rest_call'] = 'true'```
```bag_url = site_url + reverse(match.url_name, kwargs=match.kwargs)```